grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -916,14 +916,14 @@
       "steppedLine": true,
       "targets": [
         {
-          "expr": "irate(chi_clickhouse_event_SelectQuery{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
+          "expr": "rate(chi_clickhouse_event_SelectQuery{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
           "intervalFactor": 2,
           "legendFormat": "select {{hostname}}",
           "refId": "B",
           "step": 10
         },
         {
-          "expr": "irate(chi_clickhouse_event_Query{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
```

Comment:
[1m]

maybe it would be better to use $_rate_interval?

Reply:
ok. let's try

Reply:
thanks for suggestions